scripts/build/build_all_travis_container.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This honestly feels more like a Makefile to me than a bash script. But anyway, some more documentation would be nice. I have no clue what it does and why? When I would want to run it? Also the input format is underspecified.